packages/hello-gsm/src/pages/calculator/index.tsx

```diff
@@ -50,6 +51,21 @@ const getInfo = async (accessToken: string) => {
  * 둘다 없다면 로그인 페이지로 이동
  */
 export const getServerSideProps: GetServerSideProps = async ctx => {
+  // 접수 기간이 아니면 페이지 접근 불가
+  if (
+    !(
+      new Date() >= new Date('2022-10-17 9:00') &&
+      new Date() <= new Date('2022-10-20 17:00')
+    )
+  ) {
+    return {
+      props: {},
+      redirect: {
+        destination: '/404',
```

Comment:
```suggestion
        destination: '/',
```

redirect를 404가 아닌 index로 하는건 어떤가요?
404로 redirect를 시켰을 시에 user-flow가 개인적으로 부자연스러워 질 것 같은데에 반해
index로 redirect를 시켰을 경우 원서 접수 버튼이 비활성화가 되어있고, 접수 기간이 아님을 나타낼 수 있어서 의견 내봅니다!

Reply:
넵 저게 나을 것 같네요!

Reply:
> 넵 저게 나을 것 같네요!

의견 수용 감사합니다 🙇‍♂️

Reply:
이 방식으로 다른 페이지에도 추가할까요?